todo/views/team.py

```
@@ -0,0 +1,85 @@
+from rest_framework.request import Request
+from django.conf import settings
+
+from todo.serializers.create_team_serializer import CreateTeamSerializer
+from todo.services.team_service import TeamService
+from todo.dto.team_dto import CreateTeamDTO
+from todo.dto.responses.create_team_response import CreateTeamResponse
+from todo.dto.responses.error_response import ApiErrorResponse
+from todo.constants.messages import ApiErrors
+
+
+class TeamListView(APIView):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)